### DISCUSSION TITLE
Hide icon on taskbar?

### DISCUSSION THREAD

**Discussion by Imperium4** [kangyu-california/PersistentWindows#154] [Q&A] [answered]
Hello, is it possible to hide the icon on the sytem tray (right of the taskbar)? I am not talking about putting it in the container with an arrow, but removing the icon's display.
I use show all taskbar icons and I don't think I'll ever change anything of PW so I was wondering if this is possible.

**Comment by kangyu-california** [comment, marked as answer]
A new command option -nogui has been added to hide the icon, this will be available in the next release.

**Reply by kangyu-california** [reply]
Try the latest release
https://github.com/kangyu-california/PersistentWindows/releases/tag/5.35

**Comment by negoalex** [comment]
There is any setting to automaticaly send the image to the second monitor when I open the second Monitor? For example : If I opend the pc with 1 monitor, everything will be on monitor 1, when Im opening the second monitor the pages are still on monitor 1. I want when the monitor is open to automatically send the pages to second monitor. 
Thank you!!
